pkgs/applications/networking/instant-messengers/gtkcord/default.nix

```
@@ -0,0 +1,45 @@
+{ makeDesktopItem, lib, buildGoModule, fetchFromGitHub, pkg-config, glib, go, gtk3, libhandy }:
+
+buildGoModule rec {
+  pname = "gtkcord";
+  version = "0.0.4";
+  
+  src = fetchFromGitHub {
+    owner = "diamondburned";
+    repo = "gtkcord3";
+    rev = "v${version}";
+    sha256 = "0n9600s50zpx8kv89icrlxarm3633gnizg1sq1zpask8bbv2xnfd";
+  };
+  
+  nativeBuildInputs = [ pkg-config go ];
```

```suggestion
  nativeBuildInputs = [ pkg-config ];
```
go is added automatically.